client/src/components/pages/RespondentQuiz.js

```diff
@@ -16,12 +17,12 @@ const RespondentQuiz = () => {
     const navigate = useNavigate();
 
     const [questionsAnswered, setQuestionsAnswered] = useState([]);
-    const [alert, setShowAlert] = useState(false);
+    const [timeAlert, setShowTimeAlert] = useState(false);
+    const [submitAlert, setShowSubmitAlert] = useState(false);
     const [confirmMessage, setConfirmMessage] = useState('');
-    const [confirm, setConfirm] = useState(false);
 
     let time = quiz_resp && quiz_resp.timeLimit;
-    const [stateTimeLimit, setStateTimeLimit] = useState(1);
+    const [stateTimeLimit, setStateTimeLimit] = useState(2);
```

Comment:
is there a reason for changing this to 2?

Reply:
This is because on the initial render of the page if stateTimeLimit is 1 then the modal will appear displaying 1 minute left. When the component mounts, or useEffect hook renders, then stateTimeLimit changes the value to the quiz time. Because of the delays between the page rendering and the state changing the value is set to 2 instead to avoid that modal appearing.